### PR TITLE
Enable aws native fencing

### PR DESCRIPTION
**Pull request by BillAnastasiadis**
Enabling AWS native fencing.

- Related ticket: https://jira.suse.com/browse/TEAM-8702
- Verification Runs: https://openqaworker15.qa.suse.cz/tests/282115
https://openqaworker15.qa.suse.cz/tests/282304

### CODE REVIEW

File: ansible/playbooks/tasks/cluster-bootstrap.yaml [hunk 1]
```diff
@@ -317,14 +317,29 @@
...
     - is_primary
 
+- name: Configure AWS EC2 STONITH
+  ansible.builtin.command: >
+    crm configure primitive rsc_aws_stonith stonith:external/ec2
+    op start interval=0 timeout=180
+    op stop interval=0 timeout=180
+    op monitor interval=120 timeout=60
+    meta target-role=Started
+    params tag={{ aws_stonith_tag}} pcmk_delay_max=15
```
Comment (mpagot) [inline ansible/playbooks/tasks/cluster-bootstrap.yaml:327]:
```suggestion
    params tag={{ aws_stonith_tag }} pcmk_delay_max=15
```

File: terraform/aws/inventory.tmpl [hunk 2]
```diff
@@ -1,9 +1,10 @@
 all:
   vars:
     use_sbd: ${use_sbd}
-    aws_route_table: ${routetable_id}
+    aws_route_table_id: ${routetable_id}
     aws_cluster_ip: ${virtual_ip}
     aws_stonith_tag: ${stonith_tag}
+    aws_region: ${region}
```
Comment (mpagot) [inline terraform/aws/inventory.tmpl:7]: I propose to leave out `region` for the moment

File: terraform/aws/outputs.tf [hunk 3]
```diff
@@ -108,7 +108,8 @@ resource "local_file" "ansible_inventory" {
       use_sbd             = local.use_sbd,
       routetable_id       = aws_route_table.route-table.id,
       virtual_ip          = local.hana_cluster_vip,
-      stonith_tag         = module.hana_node.stonith_tag
+      stonith_tag         = module.hana_node.stonith_tag,
+      region              = var.aws_region
```
Comment (mpagot) [inline terraform/aws/outputs.tf:112]: I propose to leave out `region` for the moment

**Review by mpagot — APPROVED** [review]
LGTM

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Pull request merged**